pkgs/by-name/ch/choose-gui/package.nix

```
@@ -0,0 +1,51 @@
+{ lib, stdenv, fetchFromGitHub, xcbuild, coreutils, darwin }:
+
+stdenv.mkDerivation rec {
+  pname = "choose-gui";
+  version = "1.3.1";
+
+  src = fetchFromGitHub {
+    owner = "chipsenkbeil";
+    repo = "choose";
+    rev = version;
+    hash = "sha256-oR0GgMinKcBHaZWdE7O+mdbiLKKjkweECKbi80bjW+c=";
+  };
+
+  buildInputs = [
+    xcbuild
+    coreutils
+    darwin.apple_sdk.frameworks.Cocoa
```

```suggestion
    coreutils
    darwin.apple_sdk.frameworks.Cocoa
    xcbuild
```
(Whenever possible) follow alphabetical order.

---

Done, thanks!